src/components/AddFacilityModal/AddFacilityModal.jsx

```diff
@@ -8,12 +8,15 @@
   Button,
   useDisclosure,
   Input,
+  useToast,
+  VStack,
   // Avatar,
 } from '@chakra-ui/react';
 // import { AddIcon } from '@chakra-ui/icons';
 import React from 'react';
 import './AddFacilityModal.css';
 import UploadAvatar from '../UploadAvatar/UploadAvatar';
+import CreateToast from '../Toasts/CreateToast';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'CreateToast' is defined but never used.